src/main/java/com/gmail/dto/UserDtoImpl.java

```diff
@@ -0,0 +1,95 @@
+package com.gmail.dto;
+
+import com.gmail.entities.UserEntity;
+import com.gmail.exceptions.UserNotFoundException;
+import com.gmail.service.UserService;
+import lombok.extern.log4j.Log4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Created by Space on 17.05.2019.
+ */
+@Log4j
+@Component
+public class UserDtoImpl implements UserDto {
```

Comment:
read about what is dto in java http://qaru.site/questions/155763/dto-pattern-best-way-to-copy-properties-between-two-objects